examples/Building-and-deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb

```diff
@@ -7,7 +7,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    )
</pre><p>so we can save this as&nbsp;<code>user_attributes</code>&nbsp;if you want like below:</p><p> </p><p><code>user_features.to_parquet(</code></p><p><code>&nbsp;&nbsp;os.path.join(feature_repo_path, "data", "user_attributes.parquet")</code></p><p><code>)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/1022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>